### DISCUSSION TITLE
Not a bug just a question

### DISCUSSION THREAD

**Discussion by LeD0c4** [IzzelAliz/Arclight#502] [Q&A]
Hi, In first thank for all of us for your job !
Now i want make a version for my server but, when i compiling it, sometime (all the time but not the same ligne) my import is up to date but the other no. i dont know why

**Comment by heipiao233** [comment]
Endure it.

**Reply by SebsOficial1** [reply]
Hello im new how can i increase the ram of the server?

**Reply by Yizhouuu** [reply]
yes, config the file named like jvm_args

**Comment by heipiao233** [comment]
Install lots of optimization mods.
